plugins/remote-config/frontend/public/javascripts/countly.models.js

```diff
@@ -122,6 +122,20 @@
                             parameter.status = "Expired";
                         }
                     });
+                    if (conditions.length > 0) {
+                        var map = {};
+                        conditions.forEach(function(c) {
+                            map[c._id] = {
+                                "condition": c.condition,
+                                "condition_color": c.condition_color,
+                                "condition_definition": c.condition_definition,
+                                "condition_name": c.condition_name,
+                                "seed_value": c.seed_value,
+                                "used_in_parameters": c.used_in_parameters
+                            };
+                        });
+                        context.dispatch("countlyRemoteConfig/parameters/conditionsMap", map, {root: true});
```

Comment:
as you must have noticed in the code, for conditions we have a separate sub module.
Therefore, actions, getters, mutations related to conditions should be within conditions submodule not inside parameters submodule.

Also why create a map for conditions when you already have conditions in the vuex in an array.
You can easily use .filter to get the relevant condition from the array.
If you want to use a map instead of array due to performance issues, I suggest you keep only the map then. You can get rid of the conditions array from the vuex.